src/d3d_prefida.pro

```diff
@@ -32,27 +34,27 @@ PRO d3d_prefida, inputs, igrid=igrid,bgrid=bgrid
         for i=0,n_elements(c)-1 do begin
             spec = d3d_chords(c[i], shot=inputs.shot)
             inputs.calc_fida = 1
-            inputs.calc_bes = 1
+            inputs.calc_nbi = 1
             inputs.calc_brems = 1
         endfor
     endif
 
     w = where("npa_diag" eq strlowcase(TAG_NAMES(inputs)),nw)
     if nw ne 0 then begin
         if n_elements(inputs.npa_diag) ne 0 then begin
-            npa = d3d_npa() 
+            npa = d3d_npa()
             inputs.calc_npa_wght = 1
```

Comment:
Since we now have fast npa we should turn on `calc_npa=1` instead of `calc_npa_wght`. Also we now have the inpa geometry so you need a case statement to select the right npa chords`